### PR TITLE
fix(dop): issue route bug

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
Issue route bug.

## I have checked the following points:
- [x] I18n is finished and updated by cli
- [x] Form fields validation is added and length is limited
- [x] Display normally on small screen
- [x] Display normally when some data is empty or null
- [x] Display normally in english mode

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
✅ Yes(version is required)
hotfix/12-21

## Which issue(s) this PR fixes:

Fixes # https://erda-org.erda.cloud/erda/dop/projects/387/issues/all?id=268044&issueFilter__urlQuery=eyJzdGF0ZXMiOls0NDAyLDcxMDQsNzEwNSw0NDAzLDQ0MDQsNzEwNiw0NDA2LDQ0MDcsNDQxMiw0NTM4LDQ0MTMsNDQxNCw0NDE1LDQ0MTZdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&iterationID=680&type=BUG

### CODE REVIEW

File: shell/app/modules/project/router.tsx [hunk 1]
```diff
@@ -75,11 +78,29 @@ function getProjectRouter(): RouteConfigItem[] {
...
-              tabs: COLLABORATE_TABS,
-              ignoreTabQuery: true,
-              getComp: (cb) => cb(import('project/pages/issue/task')),
-              layout: {
-                noWrapper: true,
+              render: (props) => {
+                const { location } = props;
+                const { search } = location;
+                const params = qs.parse(search);
+                return <Redirect to={`all?${qs.stringify({ ...params, type: 'TASK' })}`} />;
```
Comment (daskyrk) [inline shell/app/modules/project/router.tsx:85]: Is this relative path ?

Comment (hujiahao-hjh) [reply shell/app/modules/project/router.tsx:85]: Yes, it should be the relative path of this route.

File: shell/app/modules/project/router.tsx [hunk 2]
```diff
@@ -75,11 +78,29 @@ function getProjectRouter(): RouteConfigItem[] {
...
+              },
+            },
+            {
+              path: 'bug',
```
Comment (daskyrk) [inline shell/app/modules/project/router.tsx:89]: path in goTo.pages should also update.

**Pull request merged**